lib/auth0/exception.rb

```diff
@@ -53,7 +53,7 @@ class InvalidApiNamespace < Auth0::Exception; end
   # Auth0 API rate-limiting encountered
   class RateLimitEncountered < Auth0::HTTPError
     def reset
-      Time.at(headers['X-RateLimit-Reset']).utc
+      Time.at(Integer(headers[:x_ratelimit_reset])).utc
```

Comment:
Any reason not to use `to_i` instead? It seems it would be more idiomatic.
It would also avoid raising an exception if the header is somehow empty.

Reply:
Hello @dmathieu

Thanks for the review.

With `Integer` I wanted to avoid `''.to_i #=> 0` and then
```
Time.at(' '.to_i)
=> 1970-01-01 01:00:00 +0100
```

If we want to accept empty header value, I think we should early return
```ruby
x_ratelimit_reset = headers[:x_ratelimit_reset]
return if x_ratelimit_reset.nil? || x_ratelimit_reset.empty?

Time.at(headers[:x_ratelimit_reset]).utc
```